test/cpp/jit/test_gpu.cpp

```diff
@@ -19072,6 +19096,9 @@ TEST(NVFuserTest, PersistentBufferProjection_CUDA) {
 }
 
 TEST(NVFuserTest, FusionIssue1223_CUDA) {
+  if (at::cuda::getDeviceProperties(0)->major < 6) {
```

Comment:
Why are this test and the others skipped?

Reply:
We should be supporting only Volta+ but I guess these tests empirically failed on the maxwell in the upstream CI